src/tests/stub_liblog.cc

```diff
@@ -23,21 +23,21 @@ AccessLogEntry::SslDetails::SslDetails() {STUB}
 #endif
 */
 void accessLogLogTo(CustomLog *, AccessLogEntry::Pointer &, ACLChecklist *) STUB
-void accessLogLog(AccessLogEntry::Pointer &, ACLChecklist *) STUB
-void accessLogRotate(void) STUB
-void accessLogClose(void) STUB
-void accessLogInit(void) STUB
-const char *accessLogTime(time_t) STUB_RETVAL(nullptr)
+    void accessLogLog(AccessLogEntry::Pointer &, ACLChecklist *) STUB
```

Comment:
Changes in this file all look very messy. Were they created by astyle?

Please remove the out of scope changes so only the new ones are visible in the merge.

Reply:
Done.

The syntax we use for STUB confuses automatic formatters.
Could we maybe mass-rename it to {STUB;} ? Definitely not in scope for this change tho

Reply:
> Could we maybe mass-rename it

I have not studied the details, but I have seen related problems in the past, and suspect that mass-renaming is probably the right long-term solution despite its pain. I think that dedicated PR should also change the `STUB` and related definitions so that we write `{ Stub(); }` or some such, confusing even fewer automation scripts _and_ allowing the compiler to detect  any forgotten `STUB` uses.